ansible/roles/apache/README.md

```diff
@@ -0,0 +1,6 @@
+### apache
```

Comment:
The `README.md` files are generated automatically from the `.rst` files when the DebOps Collection is created. You should remove this file and add the changes in the documentation, see other roles for an example.

Reply:
Ok, I'll do that a bit later